lib/rubocop/cop/performance/sum.rb

```diff
@@ -0,0 +1,135 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Performance
+      # This cop identifies places where custom array summation
```

Comment:
This description is a bit misleading as the method is not specific to Array. See https://blog.bigbinary.com/2016/11/02/ruby-2-4-introduces-enumerable-sum.html